src/guess-the-number.js

```diff
@@ -0,0 +1,26 @@
+const readlineSync = require('readline-sync')
+const prompt = readlineSync.question
+
+const guessTheNumber = guess => {
+  const randomNumber = randomNumberGenerator()
+    if ( guess > randomNumber ) {
+      console.log ('Your guess was too high!')
+    } else if ( guess < randomNumber ) {
+      console.log ('Your guess was too low!')
+    } else {
+      console.log ('You got it!')
+    }
+    console.log( 'The computer says: ' + randomNumber )
+}
+
+const randomNumberGenerator = () => {
```

Comment:
You may omit the braces and `return` keyword if you'd like (I prefer it; it feels a little more concise, and less visually noise to me).